GamesToGo.Desktop/Screens/MainMenuScreen.cs

```diff
@@ -1,10 +1,202 @@
+                                        Anchor = Anchor.TopCentre,
+                                        Origin = Anchor.TopCentre,
+                                        Position = new Vector2(0,600)
+                                    },
+                                    new BasicButton
+                                    {
+                                        Text= "Cerrar Sesión",
+                                        BackgroundColour = background.Colour,   //Color Boton userInformation
+                                        BorderColour = Color4.Black,
+                                        BorderThickness = 2f,
+                                        RelativeSizeAxes = Axes.X,
+                                        Masking=true,
+                                        Height = 40,
+                                        Anchor = Anchor.TopCentre,
+                                        Origin = Anchor.TopCentre,
+                                        Position = new Vector2(0,700)
+                                    }
+                                }
+                            },
+                            proyectCreation = new Container
+                            {
+                                RelativeSizeAxes =Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    new BasicButton
+                                    {
+                                        Text= "Crear Nuevo Proyecto",
+                                        BackgroundColour = new Color4 (145,144,144, 255),   //
+                                        BorderColour = Color4.Black,
+                                        BorderThickness = 2f,
+                                        RelativeSizeAxes = Axes.X,
+                                        Masking = true,
+                                        Height = 100,
+                                        Anchor = Anchor.BottomCentre,
+                                        Origin = Anchor.BottomCentre,
+                                    },
+                                    new FillFlowContainer
+                                    {
+                                        BorderColour = Color4.Black,
+                                        BorderThickness = 3f,
+                                        Masking = true,
+                                        Height = 650,
+                                        Width = 1000,
+                                        Anchor = Anchor.TopCentre,
+                                        Origin = Anchor.TopCentre,
+                                        Position = new Vector2(0,200),
+                                        Children = new Drawable[]
+                                        {
+                                            new ProjectDescriptionButton
+                                            {
```

Comment:
If these are going to be like this for all buttons, then apply them at component level